src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="item" data-qa="card">
+      <img src="images/imac.jpeg"
+       alt="Apple A1419"
+       class="item__image"
+       >
+      <h4 class="item__description">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h4>
+      <span class="item__number">
+        Код товара: 195434
+      </span>
+      <div class="item__rate">
+        <div class="item__stars-block">
```

Comment:
```suggestion
        <div class="item__stars-block item__stars-block--4">
```
Number of active stars should change according to the number that you write in your class (--1, --2, --3, etc.)

Reply:
Okay, all mistakes were corrected!